src/options/options.go

```diff
@@ -0,0 +1,210 @@
+package options
+
+import (
+	"fmt"
+	"github.com/iancoleman/strcase"
+	"github.com/spf13/cast"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"os"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"strings"
+)
+
+// Options contains parsed flags and ENV variables
+type Options struct {
+	Verbose          bool   `flag:"verbose"`           // verbose mode, print details to console
+	LogFilePath      string `flag:"log-file"`          // path to the log file
+	ApiHost          string `flag:"storage-api-host"`  // api host
+	ApiToken         string `flag:"storage-api-token"` // api token
+	WorkingDirectory string // working directory
+	ProjectDirectory string // project directory with ".keboola" metadata dir
```

Comment:
Project dir je adresar, ktory obsahuje zlozku `.keboola` -> s metadatami. Je to Working dir, alebo nejaky jeho nadradeny adresar (podobne ako funguje `git`).